src/windows/tools/drillresults.py

```diff
@@ -412,8 +412,10 @@ def checkreport(reportfile, crasherfile, crash_hash):
 
 
 #    pc_module = pc_in_mapped_address(reporttext, instraddr)
-    crashid['exceptions'][exceptionnum]['pcmodule'] = pc_in_mapped_address(reporttext, instraddr)
-
+    try:
+        crashid['exceptions'][exceptionnum]['pcmodule'] = pc_in_mapped_address(reporttext, instraddr)
+    except:

```

Comment:
This shouldn't catch all exceptions, it should be more specific.

Also, it's not clear whether the core problem is here or in the pc_in_mapped_address() function.